src/vivarium/framework/engine.py

```diff
@@ -360,82 +359,68 @@ class Builder:
     This is the access point for components through which they are able to
     utilize a variety of interfaces to interact with the simulation framework.
 
-    Attributes
-    ----------
-    configuration : ``LayeredConfigTree``
-        Provides access to the :ref:`configuration<configuration_concept>`
-    logging : LoggingInterface
-        Provides access to the :ref:`logging<logging_concept>` system.
-    lookup : LookupTableInterface
-        Provides access to simulant-specific data via the
-        :ref:`lookup table<lookup_concept>` abstraction.
-    value : ValuesInterface
-        Provides access to computed simulant attribute values via the
-        :ref:`value pipeline<values_concept>` system.
-    event : EventInterface
-        Provides access to event listeners utilized in the
-        :ref:`event<event_concept>` system.
-    population : PopulationInterface
-        Provides access to simulant state table via the
-        :ref:`population<population_concept>` system.
-    resources : ResourceInterface
-        Provides access to the :ref:`resource<resource_concept>` system,
-        which manages dependencies between components.
-    results : ResultsInterface
-        Provides access to the :ref:`results<results_concept>` system.
-    randomness : RandomnessInterface
-        Provides access to the :ref:`randomness<crn_concept>` system.
-    time : TimeInterface
-        Provides access to the simulation's :ref:`clock<time_concept>`.
-    components : ComponentInterface
-        Provides access to the :ref:`component management<components_concept>`
-        system, which maintains a reference to all managers and components in
-        the simulation.
-    lifecycle : LifeCycleInterface
-        Provides access to the :ref:`life-cycle<lifecycle_concept>` system,
-        which manages the simulation's execution life-cycle.
-    data : ArtifactInterface
-        Provides access to the simulation's input data housed in the
-        :ref:`data artifact<data_concept>`.
-
     Notes
     -----
     A `Builder` should never be created directly. It will automatically be
     created during the initialization of a :class:`SimulationContext`
 
     """
 
-    def __init__(self, configuration, plugin_manager):
+    def __init__(self, configuration: LayeredConfigTree, plugin_manager):
         self.configuration = configuration
+        """Provides access to the :ref:`configuration<configuration_concept>`"""
```

Comment:
This is all unrelated, but my doc-builds were failing for an unrelated reason (I forgot to add the new doc rst files), and I changed these to see if it would fix that (it didn't). 

However, I left the changes in, since I believe this is how we'd rather our docs for attributes to look.